packages/nodes-base/nodes/HttpRequest/GenericFunctions.ts

```diff
@@ -88,7 +88,14 @@ export function sanitizeUiMessage(
 			),
 		};
 	}
-
+	//redact headers with Base64 encoded credentials
+	const headers = sendRequest.headers as IDataObject;
+	if (headers) {
+		const headerKey = Object.keys(headers).find((key) => key.toLowerCase() === 'authorization');
+		if (headerKey) {
+			headers[headerKey] = REDACTED;
+		}
+	}
```

Comment:
If you want to add more headers to redact

```suggestion
	// Remove any content from blocklisted headers
	const HEADER_BLOCKLIST = new Set(['authorization']);
	if (request.headers) {
		for (const headerName of Object.keys(request.headers)) {
			if (HEADER_BLOCKLIST.has(headerName.toLowerCase())) {
				request.headers[headerName] = REDACTED;
			}
		}
	}
```